test/Refs.js

```diff
@@ -91,4 +91,18 @@ describe('Refs', () => {
 			})
 		})
 	})
+
+	it('should execute element string ref callback', async () => {
+		const target = document.createElement('div')
+		const stack = []
+
+		const {firstChild} = await render(h('input', {}), target)
+		firstChild.callback = () => stack.push(true)
```

Comment:
should also assert `event.current` set correctly

Reply:
Why?

Reply:
nvm, was confused